src/runtime/mod.rs

```diff
@@ -153,7 +154,10 @@ pub struct Runtime {
 #[derive(Debug)]
 struct Inner {
     /// A handle to one of the per-worker reactors.
```

Comment:
This comment needs to be updated:

```suggestion
     /// A handle to the reactor in the background thread.
```